active/ZEP0000.md

```diff
@@ -60,29 +64,53 @@ that require collaboration across multiple projects.
-the core specification of Zarr. The changes related to core specification follow a
+Specification ZEPs deal with the changes related to [Zarr Specifications]. The
+SPEC ZEP (abbreviation of Specification ZEP) introduces changes to the
+specification of Zarr. The changes related to specification follow a
 strict and thorough review process and should be adopted by everyone in the
-Zarr community.
+Zarr community. Specification ZEPs are of two types. They are:
 
 - **Core protocol ZEP**
 
 Describes a ZEP which involves changes in the core specification of Zarr. Core
 protocol ZEPs (commonly known as Core ZEPs) are a part of SPEC ZEPs and apply
 to [Zarr Specifications] and its various implementations under the
-[zarr-developers/zarr_implementations] GitHub repo. The core protocol ZEPs
-should be adopted by every implementation of Zarr and, in general, the overall
-Zarr community. Core ZEPs must go through a thorough review process, including
-involvement, discussion, and voting from the author of the proposed ZEP, Zarr
-Steering Council, author(s) of various Zarr implementations (ZIC), and
-open-source projects/research groups using Zarr and the general Zarr community.
-The general advice is that everyone should be made aware of changes introduced
-by core ZEPs. Core protocol ZEPs require community consensus, and developers or
-users are typically not free to ignore them.
-
-Note: Currently, Specification ZEPs only deal with the core specification of Zarr.
-The scope of SPEC ZEPs will be extended in the future.
+[zarr.dev/implementations]. The core protocol ZEPs should be adopted by every
+implementation of Zarr and, in general, the overall Zarr community. Core ZEPs
+must go through a thorough review process, including involvement, discussion,
+and voting from the author of the proposed ZEP, Zarr Steering Council, author
+(s) of various Zarr implementations (ZIC), and open-source projects/research
+groups using Zarr and the general Zarr community. The general advice is that
+everyone should be made aware of changes introduced by core ZEPs. Core protocol
+ZEPs require community consensus, and developers or users are typically not
+free to ignore them.
+
+- **Protocol extension ZEP**
+
+Describes a ZEP which involves addition to the extension of the core
+specification protocol for Zarr. Protocol extension ZEPs (commonly known as
+extension ZEPs) are part of Spec ZEPs and apply to Zarr storage extension
+specification and various implementations under the Zarr Implementations. The
+extension ZEPs are designed so that they are not necessary to be adopted by
+every other Zarr implementation. The extension ZEPs may or may not include
+involvement from the author of various Zarr implementations and the general
+Zarr community. The general idea of having an extension ZEP is to add
+functionality, data types and features to accommodate emerging and essential
+use cases.
+
+- **Lean ZEP**
```

Comment:
Can you provide an example for a Lean ZEP? Where would adding a codec fall?

Reply:
I agree with this comment. In particular, how will an author choose between an Extension ZEP and an Lean ZEP?

Reply:
Please see: https://github.com/zarr-developers/zeps/pull/59#issuecomment-1989703623.